carbon/db.py

```diff
@@ -17,9 +17,17 @@
                 Column('EMAIL_ADDRESS', String),
                 Column('ORIGINAL_HIRE_DATE', Date),
                 Column('APPOINTMENT_END_DATE', Date),
-                Column('HR_ORG_UNIT_TITLE', String),
                 Column('PERSONNEL_SUBAREA_CODE', String),
-                Column('JOB_TITLE', String))
+                Column('JOB_TITLE', String),
+                Column('HR_ORG_UNIT_ID', String),
+                )
+
+dlcs = Table('HR_ORG_UNIT', metadata,
+             Column('HR_ORG_UNIT_ID', String,
+                    ForeignKey('HR_ORG_UNIT.HR_ORG_UNIT_ID')),
```

Comment:
This looks a little weird. This is a foreign key to itself. Is this intentional?